instructure/canvas-ios
# Refactor Page Details

refs: MBL-12129
affects: student, teacher
release note: none

Test Plan:
 - Make sure the experimental feature flags are turned on
 - Go to pages list and create a page
 - It should show up in the list
 - Tap on a page
 - Edit the page including the title and the details and hit done
 - The detail screen should update to reflect the changes
 - Hit back and the pages list should have the new title and update date
 - Tap on a page
 - Make it the front page
 - Make sure that is reflected on the pages list
 - Delete a page
 - It should remove the page from pages list and pop the details VC
 - Links to canvas other routable canvas content should work
 - Links to safari should work
 - Pull to refresh should work
 - Pages in groups should work
